qa/os/src/test/java/org/elasticsearch/packaging/test/CertGenCliTests.java

```diff
@@ -48,6 +50,8 @@ public void test10Install() throws Exception {
         install();
         // Enable security for this test only where it is necessary, until we can enable it for all
         ServerUtils.enableSecurityFeatures(installation);
+        // Disable security auto-configuration as we want to generate keys/certificates manually here
+        ServerUtils.disableSecurityAutoConfiguration(installation);
```

Comment:
Do we need to invoke `ServerUtils.enableSecurityFeatures(installation)` and `ServerUtils.disableSecurityAutoConfiguration(installation)`  here?

I think the `CertGenCliTests` should work irrespective if the node has been started (or still running), and if it has, whether it had been auto-configured.

PS I'm having trouble running these tests but maybe you hacked it on your linux.

Reply:
> Do we need to invoke ServerUtils.enableSecurityFeatures(installation) and ServerUtils.disableSecurityAutoConfiguration(installation) here?

`ServerUtils.enableSecurityFeatures(installation)` only applies to packaged installations ( where we still disable security after installation) and will be removed with https://github.com/elastic/elasticsearch/pull/75144. I thought I had added a comment about it but I probably didn't :) 

We don't **need** `ServerUtils.disableSecurityAutoConfiguration(installation)` but then again we don't need the auto-configured TLS setup as the point of this test is to use TLS material that is generated with CertificateTool

Reply:
> PS I'm having trouble running these tests but maybe you hacked it on your linux.

Yes, I can run these locally and this has saved me tons of time :)

Reply:
Thanks for the explication, I don't have further questions or suggestions here.